src/scripts/main.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+const largeImg = document.querySelector('#largeImg');
+const list = document.querySelector('.gallery__list');
+
+list.addEventListener('click', (someEvent) => {
+  const link = someEvent.target.closest('.list-item__link');
+  const srcImg = link.href;
```

Comment:
You should check if `link` is not null before accessing its `href` property. Consider moving the `const srcImg = link.href;` line after the null check to avoid potential errors if `link` is null.